src/mono/wasm/wasm.proj

```diff
@@ -104,7 +104,9 @@
                        $(NativeBinDir)dotnet.wasm;
                        $(NativeBinDir)dotnet.timezones.blat;
                        $(NativeBinDir)libicuuc.a;
-                       $(NativeBinDir)libicui18n.a"
+                       $(NativeBinDir)libicui18n.a;
+                       $(NativeBinDir)libSystem.Native.a;
```

Comment:
Shouldn't that already be copied over as part of the native build?

Reply:
Yeah they're already there in my local build and in the runtime pack produced by CI. Is that not the case for you? Note that you need to build the `libs` subset.

Reply:
I'm running make runtime in src/mono/wasm, and that one copies everything but these 2 files.

Reply:
yeah that's expected, the System.Native build is done as part of the libs.native subset, we should add that to the `make runtime` target instead of this.